bin/test.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Test runner for Flex Plugin Builder
+ * This is invoked via `npm run test`. You can pass additional parameters using `npm run test -- param1 param2`
+ * To run a single test, use `npm run test:one packages/path/to/file/or/directory
+ */
+
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+const os = require('os');
+
+// These packages are ignored and no jest runs for them
+const ignorePackages = ['flex-plugin-e2e-tests'];
+
+// fs helper commands
+const isWin = () => os.platform() === 'win32';
+const exists = (src) => src && fs.existsSync(src);
+const isDirectory = (src) => exists(src) && fs.lstatSync(src).isDirectory();
+const isFile = (src) => exists(src) && fs.lstatSync(src).isFile();
+const isEmpty = (src) => exists(src) && isDirectory(src) && fs.readdirSync(src).length !== 0;
+
+// Promisified spawn command
+const spawn = async (cmd, args) => {
+  return new Promise((resolve) => {
+    const shell = isWin() ? true : process.env.SHELL;
+
+    const child = cp.spawn(cmd, args, {
+      cwd: process.cwd(),
+      env: process.env,
+      shell,
+      stdio: 'inherit',
+      encoding: 'utf-8',
+    });
+    child.on('exit', (code) => {
+      if (code !== 0) {
+        process.exit(code);
```

Comment:
Can you add a print statement to confirm that cmd failed with non zero exit code?